tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+
+
+*** Test Cases ***
+Verify Pytorch And Tensorflow Can Be Spawned
+    [Documentation]    Verify Pytorch and tensorflow can be spawned
+    [Tags]    Sanity
+    ...       RHODS-1469
```

Comment:
you'd set the polarion ID (e.g., ODS-XYZ) as tag instead of Jira (i.e., RHODS-1469)

Reply:
Updated